jobserver/authorization/roles.py

```diff
@@ -47,6 +56,20 @@ class OnboardingAgent:
     delivering a project, to a degree that has been pre-planned and resourced.
     """
 
+    models = [
+        "jobserver.models.User",
+    ]
+    permissions = []
+
+
+class OrgCoordinator:
+    """
+    The responsible party for an Org
```

Comment:
The definite article in the docstring suggests that there might only ever be one `OrgCoordinator` for a given organisation -- is this true?  If not, maybe change `The` to `A`?